paint-timing/fcp-only/fcp-with-rtl.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<head>
+<title>Performance Paint Timing Test: FCP due to direction change</title>
+<style>
+  #text {
+    right: -100px;
+    position: absolute;
+  }
+
+  body {
+    direction: rtl;
+  }
+</style>
+</head>
+<body direction="RTL">
```

Comment:
Is this and the style above both needed?

Reply:
Oops, will post a subsequent patch.